libraries/botframework-connector/src/auth/passwordServiceClientCredentialFactory.ts

```diff
@@ -42,31 +42,27 @@ export class PasswordServiceClientCredentialFactory implements ServiceClientCred
         loginEndpoint: string,
         validateAuthority: boolean
     ): Promise<ServiceClientCredentials> {
-        if (!(await this.isValidAppId(appId))) {
```

Comment:
@stevengum I noticed this was missing [in .NET](https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Connector/Authentication/PasswordServiceClientCredentialFactory.cs#L76). This was causing anonymous parent -> skill failures. Any idea what's going on?

Reply:
Tracked via https://github.com/microsoft/botbuilder-dotnet/issues/5591 in .NET repo.